.github/workflows/common.yml

```diff
@@ -53,7 +53,7 @@ jobs:
           sudo apt-get autoremove
           sudo apt-get autoclean
           pip install --user --upgrade setuptools
-          pip install tox==4.6.3
+          pip install 'tox>=4.21,<5' tox-uv
```

Comment:
Consider adding a comment explaining why tox-uv is being used and its benefits for future maintainers. For example:
```yaml
# Using tox-uv for faster dependency installation in CI
# Minimum tox version 4.21 required for tox-uv compatibility
```